jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/params/BoostingAlgorithmParams.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ Copyright (c) 2014 by Contributors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+package ml.dmlc.xgboost4j.scala.spark.params
+
+import scala.collection.immutable.HashSet
+
+import org.apache.spark.ml.param._
+
+trait BoostingAlgorithmParams extends Params {
+
+  /**
+   * The number of rounds for boosting
+   */
+  val round = new IntParam(this, "num_round", "The number of rounds for boosting",
```

Comment:
I wish we could rename it to numRound...

I actually have a branch doing that, but the diff is huge.

Reply:
Yeah, `numIterations` or `numRounds` sounds better to me, but it would break compatibility